### PR TITLE
Add the HW requirements for running a node

### PR DESCRIPTION
**Pull request by andreipope**
Add the HW requirements for running a node

### CODE REVIEW

File: docs/en/jump-start-extdev-plasma-us1.md [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
 ---
 
 ## Purpose
 
 This page shows you how to launch an extdev-plasma-us1 node from the jump start.
 
-## Download stable version of loom
+## Hardware Requirements
+
+The minimum hardware requirements are as follows: 2 CPU, 16 GB RAM, NVMe SSD. Example: `i3.large`.
```
Comment (enlight) [inline docs/en/jump-start-extdev-plasma-us1.md]: `i3.large` -> `AWS i3.large`

File: docs/en/jump-start-plasma.md [hunk 2]
```diff
@@ -4,16 +4,22 @@ title: Basechain Jump Start
...
 
-Follow the instructions below to launch a Basechain node from the jump start.
+## Purpose
 
-## Download stable version of Basechain
+This page shows you how to launch a Basechain node from the jump start.
+
+## Hardware Requirements
+
+The minimum hardware requirements are as follows: 4 CPU, 32 GB RAM, NVMe SSD. Example: `i3.xlarge`.
```
Comment (enlight) [inline docs/en/jump-start-plasma.md]: `i3.xlarge` -> `AWS i3.xlarge`

File: docs/en/jump-start-extdev-plasma-us1.md (hunk 1 above)
Comment (andreipope) [reply docs/en/jump-start-extdev-plasma-us1.md]: @enlight Fixed. Thanks!

File: docs/en/jump-start-plasma.md (hunk 2 above)
Comment (andreipope) [reply docs/en/jump-start-plasma.md]: @enlight Fixed. Thanks!

**Pull request merged**